launch/launch/__init__.py

```diff
@@ -75,8 +78,11 @@ def __init__(self, cmd, name, output_handler, exit_handler, env=None):
 
     def send_signal(self, signal):
         if self.transport:
+            self.task_state.signals_received.append(signal)
             self.transport.send_signal(signal)
 
     def terminate(self):
-        if self.transport:
+        if os.name != 'nt':
+            self.send_signal(signal.SIGINT)
```

Comment:
In `launcher.py` the launcher calls `close_process` on all the exited future, which calls `Transport.close()` on the process, which send SIGTERM to the process if it hasn't exited yet.

https://github.com/ros2/launch/blob/master/launch/launch/launcher.py#L183
https://docs.python.org/3/library/asyncio-protocol.html#asyncio.BaseSubprocessTransport.close